.github/issue_template/documentation.md

```diff
@@ -0,0 +1,9 @@
+---
+name: 📚 Documentation
+about: Report an issue related to documentation
```

Comment:
Maybe "Report a problem with the documentation". But again, might as well leave blank.